src/book/chapter.rs

```diff
@@ -1,24 +1,44 @@
-use std::path;
+use std::path::{Path, PathBuf};
+use book::metadata::Author;
 
 /// The Chapter struct holds the title of the chapter as written in the SUMMARY.md file,
 /// the location of the markdown file containing the content and eventually sub-chapters
+
+/// TODO use in template: author, description, index, class
+
 #[derive(Debug, Clone)]
 pub struct Chapter {
+    /// The title of the chapter.
     title: String,
-    file: path::PathBuf,
+    /// Path to chapter's markdown file.
+    file: PathBuf,
+
+    /// TODO The author of the chapter, or the book.
+    author: Author,
```

Comment:
Maybe this should be an `Option`? Or a `Vec` because multiple authors are possible?
Can you maybe explain shortly how you see this author property interact with the author we store in the book?